tests/bgp_commands_test.py

```diff
@@ -97,6 +97,14 @@
 Error: bgp summary from bgp container not in json format
 """
 
+show_error_no_v6_neighbor = """\
+% No IPv6 neighbor is configured
+"""
+
+show_error_no_v4_neighbor = """\
+% No IPv4 neighbor is configured
```

Comment:
Please remove "% " (% and the space after it) as that is not in the original error message or the service that needs to parse it will not match...
Thanks!

Reply:
Fixed